data.json

```diff
@@ -454,5 +448,11 @@
     "githubUsername": "ritik307",
     "imageUrl": "https://user-images.githubusercontent.com/22374829/104178614-a8acea80-5430-11eb-8ee8-1a7bb89248b2.png",
     "issueNumber": 237
+  },
+  {
+    "name": "Douglas Franklin",
+    "githubUsername": "idamezhim",
+    "imageUrl": "https://user-images.githubusercontent.com/70078201/104479251-7f709380-55c3-11eb-9b28-4cd3bf56f7df.png",
+    "issueNumber": 241
   }
-]
+]
```

Comment:
Please add a new line to the end of the file.

Reply:
Thanks, Allan, I just added that now

Reply:
Looks like the same issue. Please add the new line after the `]`

Reply:
I just resolved, Thanks Allan